src/back-end/lib/routers/auth.ts

```diff
@@ -188,16 +188,67 @@ async function makeRouter(connection: Connection): Promise<Router<any, any, any,
         }
       })
     }] : []),
+    // /auth/createsessionadmin is for testing only, so only exists outside of production
+    // @ts-ignore
```

Comment:
@BCerki why do we need all of these `@ts-ignore`s?

Reply:
I was able to clear a few of them since we've updated TS. The remaining ones are because I'm faking the login, so my endpoints don't return the same thing as real auth endpoints. There some further conversation about leaving it for tech later here: https://github.com/button-inc/digital_marketplace/pull/111#pullrequestreview-838771199